File: ClientApp/src/app/interfaces/security/response/managementUsersResponse.ts
import { ByBaseResponse } from "../../by-base-response";
import { UserRoleResponse } from "./user-role-response";

export interface ManagementUsersResponse extends ByBaseResponse {
	id: string;
	fullName: string;
	email: string;
	userName: string;
	age: number;
	birthDate: string;
	phoneNumber: string;
	emailConfirmed: boolean;
	lockoutEnabled: boolean;
	lockoutEnd: string | null;
	userRoles: UserRoleResponse[];
}

File: Contracts/Model/Security/Response/ManagementUsersResponse.cs
namespace Contracts.Model.Security.Response;
public class ManagementUsersResponse : ByBaseResponse
{
	public Guid Id { get; set; }
	public string UserName { get; set; }
	public string FullName { get; set; }
	public string Email { get; set; }
	public DateOnly BirthDate { get; set; }
	public bool EmailConfirmed { get; set; }
	public DateTimeOffset? LockoutEnd { get; set; }
	public IEnumerable<UserRoleResponse> UserRoles { get; set; }
}
